src/prommis/solid_handling/crusher.py

```diff
@@ -0,0 +1,216 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES).
+#
+# Copyright (c) 2018-2023 by the software owners: The Regents of the
+# University of California, through Lawrence Berkeley National Laboratory,
+# National Technology & Engineering Solutions of Sandia, LLC, Carnegie Mellon
+# University, West Virginia University Research Corporation, et al.
+# All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
+# for full copyright and license information.
+#################################################################################
+r"""
+Crusher 
+=======
+
+Author: Lingyan Deng
+
+The Crusher module including power consumption for solid crushing. It is a function of particle size distribution, mass flow rate, and bond work index.
+
+Degrees of Freedom
+------------------
+
+A Crusher module has one degree of freedom, which is the "work" output. 
+
+Model Structure
+---------------
+
+The Crusher model mainly use port for properties goes in and out. The properties of the Crusher Unit model is mainly the particle size distribution. 
+
+Additional Constraints
+----------------------
+
+Crusher add one additional constraint to calculate the work required to crush the particles.
+
+.. math:: work_{t} = 10 \times flow_mass_in_{t} \times bond_work_index \times \left(\frac{1}{\sqrt{prod\_p80}}_{t} - \frac{1}{\sqrt{feed\_p80}}_{t}\right) 
+
+where :math: `work_{t}` is the work required to crush the particles at `t` time, 10 is an empirical value and should not be changed, :math: `flow_mass_in_{t}` is the inlet mass flow rate
+at `t` time, :math: `bond_work_index` is the bond work index of particles, :math: `prod\_p80_{t}` is production particle size with 80% passing the mesh at `t` time, :math: `feed\_p80_{t}` is 
```

Comment:
You need to include a section on Expressions which shows how the P80's are calculated.